Makefile

```diff
@@ -196,19 +196,12 @@ godocs:
 build-docs:
 	@cd docs && \
 	while read -r branch path_prefix; do \
-		(git checkout $${branch} && npm install && VUEPRESS_BASE="/$${path_prefix}/" npm run build) ; \
+		(git checkout $${branch} && npm install && VUEPRESS_BASE="/cosmos-sdk/$${path_prefix}/" npm run build) ; \
```

Comment:
Not sure about adding this`cosmos-sdk` prefix. Instead, we should add a file named `CNAME` to the `gh-pages` branch with the domain we will be using (`docs.cosmos.network`).

I know it wont work under `cosmos.github.io` domain, but will under `docs.cosmos.network` which is what we want.

Reply:
Oh okay. We have the prefix on Tendermint with the CNAME so thought it would be the same. Will add the CNAME and ask the url to be changed

Reply:
It was there at first, but it was removed later for that exact reason.

Thanks 👍  🙏

Reply:
removed the prefix and pushed CNAME to gaia and cosmos-sdk gh-pages branches.